app/src/main/java/org/simple/clinic/summary/PatientSummaryUpdate.kt

```diff
@@ -66,7 +66,7 @@ class PatientSummaryUpdate : Update<PatientSummaryModel, PatientSummaryEvent, Pa
   private fun currentFacilityLoaded(model: PatientSummaryModel, event: CurrentFacilityLoaded): Next<PatientSummaryModel, PatientSummaryEffect> {
     val updatedModel = model.currentFacilityLoaded(event.facility)
 
-    return if (event.facility.config.teleconsultationEnabled == true) {
+    return if (event.facility.config.teleconsultationEnabled == true && model.isUserLoggedIn) {
```

Comment:
We could read this property (`teleconsultationEnabled`) from `updatedModel` instead of the event.

Reply:
Ah yes, I completely forgot about that.